app/controllers/application_controller/ci_processing.rb

```diff
@@ -203,7 +203,6 @@ def ownership_update
   # Retire 1 or more items (vms, stacks, services)
   def retirevms
     assert_privileges(params[:pressed])
-    vms = find_checked_items
```

Comment:
We made a merge from ManageIQ/master and this variable remained, this was blocking Travis because it's an unused variable.